Jenkinsfile

```diff
@@ -281,6 +281,17 @@ try {
         }
       }
     },
+    'GPU: MKLDNN_CUDNNOFF': {
+       node('mxnetlinux-cpu') {
+         ws('workspace/build-mkldnn-gpu-nocudnn') {
+            timeout(time: max_time, unit: 'MINUTES') {
+               init_git()
+               docker_run('ubuntu_build_cuda', 'build_ubuntu_gpu_mkldnn_cuda', false)
```

Comment:
supposed to be build_ubuntu_gpu_mkldnn_nocudnn?

Reply:
good catch! will fix.